### PR TITLE
prepare release 3.0.0

### PR DESCRIPTION
**Pull request by mandar242**
##### SUMMARY
- Run `antsibull-changelog` to generate release notes.
- Update `galaxy.yml` version to `3.0.0`

##### ISSUE TYPE
- Release Pull Request

### CODE REVIEW

**Review by hakbailey — APPROVED** [review]
We might want to change the branch name to `stable-3.x` but otherwise this looks good to me.

**Comment by alinabuzachis** [comment]
Can you please make sure that we are following the same branch naming as in the other collections and that the links to the documentation are updated to point to stable-3 rather than main before releasing 3.0?

**Comment by mandar242** [comment]
> Can you please make sure that we are following the same branch naming as in the other collections and that the links to the documentation are updated to point to stable-3 rather than main before releasing 3.0?

@alinabuzachis @hakbailey  to kept branch name what naming was present in this repo
https://github.com/redhat-cop/cloud.aws_ops/branches/all?query=stable i.e. stable-2.0, stable-3.0
should I change it to stable-3?

File: README.md [hunk 1]
```diff
@@ -146,12 +146,12 @@ If you want to run cloud integration tests, ensure you log in to the cloud:
   ansible-test integration [target]
 ```
 
-This collection is tested using GitHub Actions. To know more about CI, refer to [CI.md](https://github.com/https://github.com/redhat-cop/cloud.aws_ops/blob/main/CI.md).
+This collection is tested using GitHub Actions. To know more about CI, refer to [CI.md](https://github.com/https://github.com/redhat-cop/cloud.aws_ops/stable-3/main/CI.md).
```
Comment (GomathiselviS) [inline README.md:149]:
```suggestion
This collection is tested using GitHub Actions. To know more about CI, refer to [CI.md](https://github.com/https://github.com/redhat-cop/cloud.aws_ops/blob/stable-3/CI.md).
```

File: README.md [hunk 2]
```diff
@@ -146,12 +146,12 @@ If you want to run cloud integration tests, ensure you log in to the cloud:
...
 
 ## Contributing to this collection
 
 We welcome community contributions to this collection. If you find problems, please open an issue or create a PR against this collection repository.
-See [CONTRIBUTING.md](https://github.com/redhat-cop/cloud.aws_ops/blob/main/CONTRIBUTING.md) for more details.
+See [CONTRIBUTING.md](https://github.com/redhat-cop/cloud.aws_ops/stable-3/main/CONTRIBUTING.md) for more details.
```
Comment (GomathiselviS) [inline README.md:154]:
```suggestion
See [CONTRIBUTING.md](https://github.com/redhat-cop/cloud.aws_ops/blob/stable-3/CONTRIBUTING.md) for more details.
```

File: README.md [hunk 3]
```diff
@@ -163,7 +163,7 @@ If you encounter issues or have questions, you can submit a support request thro
 
 ## Release Notes
 
-See the [raw generated changelog](https://github.com/redhat-cop/cloud.aws_ops/blob/main/CHANGELOG.rst).
+See the [raw generated changelog](https://github.com/redhat-cop/cloud.aws_ops/stable-3/main/CHANGELOG.rst).
```
Comment (GomathiselviS) [inline README.md:166]:
```suggestion
See the [raw generated changelog](https://github.com/redhat-cop/cloud.aws_ops/blob/stable-3/CHANGELOG.rst).
```

File: changelogs/changelog.yaml [hunk 4]
```diff
@@ -198,3 +198,30 @@ releases:
...
+        < 2.15.
+    fragments:
+    - 114-bump-ansible-lint-version.yml
+    - 20231219-deploy_flask_app-update-arguments-spec.yml
+    - 3.0.0.yml
+    - backup_select_resource_bug_fix.yml
+    - doc_link_update.yml
+    - fix_rds_db_version.yml
+    - readme_template_update.yml
+    release_date: '2024-07-29'
```
Comment (GomathiselviS) [inline changelogs/changelog.yaml:227]: Date needs to be updated

File: roles/awsconfig_multiregion_cloudtrail/README.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ License
 
 GNU General Public License v3.0 or later
 
-See [LICENSE](https://github.com/ansible-collections/cloud.aws_ops/blob/main/LICENSE) to see the full text.
+See [LICENSE](https://github.com/ansible-collections/cloud.aws_ops/stable-3/main/LICENSE) to see the full text.
```
Comment (GomathiselviS) [inline roles/awsconfig_multiregion_cloudtrail/README.md:51]:
```suggestion
See [LICENSE](https://github.com/ansible-collections/cloud.aws_ops/blob/stable-3/LICENSE) to see the full text.
```

**Review by GomathiselviS — CHANGES_REQUESTED** [review]

**Review by GomathiselviS — APPROVED** [review]

**Pull request merged**